docs/overview/08-important-links.md

```diff
@@ -1,11 +1,7 @@
 ---
-title: zkVerify Hub
+title: Resources
```

Comment:
Perhaps we can put this file in the bar on top of the page as well ? In a separate section ?